### DISCUSSION TITLE
Can we have longer than 8 character names for switch names

### DISCUSSION THREAD

**Discussion by buckeyebobm** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#124] [Feature Requests & Ideas]
I'd like to be able to have longer switch names so I can label primary and secondary actions clearly. Like AC30/Deluxe. or Tap/Tuner.

**Comment by TheGuacamoleXplosion** [comment]
yep, currently it's not even 8, it's seven. Since the switches can have double functionality, it makes sense to put those in the names. For looper control, stuff like "stop/undo" is useful. Display looks like there would be enough space for some more characters.

**Comment by simonaglover95** [comment]
@TheGuacamoleXplosion - can you send some screenshots of an 8-character label being cut off on your BRIDGE screen? The devices are supposed to display 8-character labels. 16 character bank names. 

@buckeyebobm 
At the moment we're not going to be changing the label character count. You can use "Tap/Tune" or "Tap/Tunr" & "AC30/Dlx" in the current state of things. I don't think that's unreasonable. Perhaps you wouldn't find it annoying but many people would prefer to have reasonable space between their switch labels so they're easily readable. Reducing the space makes it slower to read for most people.

**Reply by TheGuacamoleXplosion** [reply]
@simonaglover95 @buckeyebobm 
Sorry, I had forgotten how to count for a moment, "Stop/Undo" would be 9 characters :-D
So, both of you are perfectly right, current footswitch name limit is 8, not 7.
